Sent2Vec Tests and Tutorial.ipynb

```diff
@@ -0,0 +1,3008 @@
+    "\n",
+    "The sentence embedding is defined as the average of the source word embeddings of its constituent words. This model is furthermore augmented by also learning source embeddings for not only unigrams but also n-grams present in each sentence, and averaging the n-gram embeddings along with the words"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Training models"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the following examples, we'll use the Lee Corpus (which you already have if you've installed gensim) for training our model. All models are trained with the same hyperparameters for evaluation purposes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "No handlers could be found for logger \"gensim.models.fasttext\"\n",
+      "Using TensorFlow backend.\n",
+      "/Users/prerna135/anaconda/envs/gensim_env/lib/python2.7/site-packages/sklearn/cross_validation.py:44: DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.\n",
+      "  \"This module will be removed in 0.20.\", DeprecationWarning)\n"
+     ]
+    }
+   ],
+   "source": [
+    "import gensim\n",
+    "import os\n",
+    "from gensim.models.word2vec import LineSentence\n",
+    "from gensim.models.sent2vec import Sent2Vec as s2v\n",
+    "from gensim.models.fasttext import FastText as ft\n",
+    "from gensim.models.sent2vec import TorontoCorpus\n",
+    "from gensim.utils import tokenize\n",
+    "import scipy\n",
+    "import re\n",
+    "from numpy import dot\n",
+    "from gensim import matutils\n",
+    "import time\n",
+    "import numpy as np\n",
+    "import tensorflow as tf\n",
+    "import random\n",
+    "import eval_sick\n",
+    "import eval_classification\n",
+    "import eval_trec\n",
+    "import smart_open"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Prepare training data\n",
```

Comment:
Use markdown cell as headers before code (not `#` on head of cells) for big pieces of code